vcf_annotation_tools/vcf_expression_annotator.py

```diff
@@ -159,6 +166,11 @@ def define_parser():
         help="Path to write the output VCF file. If not provided, the output VCF file will be "
             +"written next to the input VCF file with a .tx.vcf or .gx.vcf file ending."
     )
+    parser.add_argument(
+        "--ignore-transcript-version",
+        help="Only match on the Ensembl transcript ID without regards to the version.",
```

Comment:
Suggested change: "Assumes that the final period and number denotes the transcript version and ignores it (i.e. for "ENST00001234.3" - ignores the ".3")"